cloudbuild.yaml

```diff
@@ -25,81 +28,299 @@ steps:
+    docker tag gcr.io/cloud-marketplace-tools/k8s/deployer_helm:$$MARKETPLACE_TOOLS_TAG \
+        gcr.io/cloud-marketplace-tools/k8s/deployer_helm:local
+  waitFor:
+  - '-'
+
+- id: Pull cloud-marketplace-tools dev Image
+  name: gcr.io/cloud-builders/docker
+  entrypoint: /bin/bash
+  args:
+  - -exc
+  - |
+    MARKETPLACE_TOOLS_TAG=sha_7c70bcd83a8c490a11f93abfc4ec7a5f39c61ab6
+
+    docker pull gcr.io/cloud-marketplace-tools/k8s/dev:$$MARKETPLACE_TOOLS_TAG
+    docker tag gcr.io/cloud-marketplace-tools/k8s/dev:$$MARKETPLACE_TOOLS_TAG \
+        gcr.io/cloud-marketplace-tools/k8s/dev:local
+  waitFor:
+  - '-'
+
+#############
+# Wordpress #
+#############
+
+- id: Build Wordpress Deployer
+  name: gcr.io/cloud-builders/docker
+  dir: wordpress
+  entrypoint: deployer/build
+  args:
+  - --project_id=$PROJECT_ID
+  - --tag=$BUILD_ID
+  waitFor:
+  - Pull cloud-marketplace-tools deployer_envsubst Image
+
+- id: Build Wordpress Tester
+  name: gcr.io/cloud-builders/docker
+  dir: wordpress
+  entrypoint: tester/build
+  args:
+  - --project_id=$PROJECT_ID
+  - --tag=$BUILD_ID
+  waitFor:
+  - '-'
+
+- id: Build Wordpress Image
+  name: gcr.io/cloud-builders/docker
+  dir: wordpress
+  entrypoint: wordpress/build
+  args:
+  - --project_id=$PROJECT_ID
+  - --tag=$BUILD_ID
+  waitFor:
+  - '-'
+
+- id: Build Wordpress Init
+  name: gcr.io/cloud-builders/docker
+  dir: wordpress
+  entrypoint: init/build
+  args:
+  - --project_id=$PROJECT_ID
+  - --tag=$BUILD_ID
+  waitFor:
+  - '-'
+
+- id: Build Wordpress Mysql
+  name: gcr.io/cloud-builders/docker
+  dir: wordpress
+  entrypoint: mysql/build
+  args:
+  - --project_id=$PROJECT_ID
+  - --tag=$BUILD_ID
+  waitFor:
+  - '-'
+
+- id: Build Wordpress Ubbagent
+  name: gcr.io/cloud-builders/docker
+  dir: wordpress
+  entrypoint: ubbagent/build
+  args:
+  - --project_id=$PROJECT_ID
+  - --tag=$BUILD_ID
+  waitFor:
+  - Initialize git
 
 - id: Fetch Wordpress Reporting Secret
   name: gcr.io/cloud-builders/gsutil
   args:
   - cp
   - gs://cloud-marketplace-ops-test-kokoro/reporting_secrets/wordpress.yaml
   - reporting-secret.json
-
-- id: Build Wordpress
-  name: gcr.io/cloud-marketplace-tools/k8s/dev:latest
   waitFor:
-  - Initialized
-  dir: wordpress
-  args:
-  - -exc
-  - make -j4 clean app/build
+  - '-'
 
 - id: Verify Wordpress
-  name: gcr.io/cloud-marketplace-tools/k8s/dev:latest
-  waitFor:
-  - Initialized
-  - Build Wordpress
-  - Fetch Wordpress Reporting Secret
+  name: gcr.io/cloud-marketplace-tools/k8s/dev:local
   env:
-  - APP_DEPLOYER=gcr.io/cloud-marketplace-ops-test/example/wordpress/deployer:latest
+  - APP_DEPLOYER=gcr.io/$PROJECT_ID/example/wordpress/deployer:$BUILD_ID
   - |
     APP_PARAMETERS={
-                     "reportingSecret": "file://reporting-secret.json",
-                     "imageTester": "gcr.io/cloud-marketplace-ops-test/example/wordpress/tester:latest"
+                     "reportingSecret": "file://reporting-secret.json"
```

Comment:
we discussed about having APP_PARAMETERS as a separate file that gets read in. Is it still the plan?